backstory/pom.xml

```diff
@@ -50,10 +50,10 @@
       <scope>test</scope>
     </dependency>
 
-    <!-- https://mvnrepository.com/artifact/org.mockito/mockito-inline -->
+    <!-- https://mvnrepository.com/artifact/org.mockito/mockito-core -->
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-inline</artifactId>
+      <artifactId>mockito-core</artifactId>
```

Comment:
The test failure is due to core not supporting mocking final classes.

If you want to use core and mock final classes, you can explicitly enable that feature by including `mock-maker-inline` as done here:
https://github.com/ykjchen/step/pull/1/files

Reply:
I will add that. Thanks!